pkg/schedule/placement/rule_manager.go

```diff
@@ -56,9 +57,10 @@ type RuleManager struct {
 	ctx     context.Context
 	storage endpoint.RuleStorage
 	syncutil.RWMutex
-	initialized bool
-	ruleConfig  *ruleConfig
-	ruleList    ruleList
+	initialized    bool
+	inMicroService bool
```

Comment:
How about skipInit?

Reply:
how about `skipLoadRule`? Otherwise. it will look weird
```
c.ruleManager.SkipIntialize()
err = c.ruleManager.Initialize(persistConfig.GetMaxReplicas(), persistConfig.GetLocationLabels(), persistConfig.GetIsolationLevel())
```

Reply:
LGTM